src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -165,9 +166,21 @@ public String retrievePrevDefault(Mod mod) {
         }
 
         @Override
-        public void setDefaultGroup(Mod mod, String value) {
+        public void setDefaultGroup(Mod mod, String value) {}
 
+        @Override
+        public void undoContents() {}
+
+        @Override
+        public boolean isEarliestContentVersion() {
+            return false;
         }
+
+        @Override
+        public VersionedContentList getVersionedContents() {
+            return null;
+        }
+
```

Comment:
Should these methods perform an assertion that they should not be called instead?

Reply:
Yup I agree with you, have changed these methods to throw the 'not supposed to be called' assertion